src/index.ts

```diff
@@ -5,4 +5,5 @@ yargs
   .commandDir("commands", { extensions: ["ts"] })
   .demandCommand()
   .strict()
+  .version(false)
```

Comment:
what's this ?

Reply:
remove printing of version, it automatically gives you a --version flag otherwise which prints the useless version from package.json which doesnt change since we are using semantic-release

Reply:
I think it should work on the final build because semantic set correctly the version before release, then reset it to sth dumb.

I will try

Reply:
it was not working before because the version was loaded from package.json. There were an implicit require which in turn was bundling the file directly before being parsed by semantic-release.

It's not the case anymore, and I guess the version is loaded from the package.json which is tar-balled, and in this one the version is correct => I think it will work for the next release.

One thing I noticed, the path to the bin in package.json is not valid anymore (src must be removed):
```json
"bin": {
    "open-attestation": "dist/cjs/index.js",
    "oa": "dist/cjs/index.js"
  },
```